vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -140,7 +142,7 @@ func (calc *UpdatePriorityCalculator) AddPod(pod *apiv1.Pod, now time.Time) {
 		klog.V(4).Infof("not updating pod %v/%v because resource would not change", pod.Namespace, pod.Name)
 		return
 	}
-	klog.V(2).Infof("pod accepted for update %v/%v with priority %v", pod.Namespace, pod.Name, updatePriority.ResourceDiff)
+	klog.V(2).Infof("pod accepted for update %v/%v with priority %v - processed recommendations:\n%v", pod.Namespace, pod.Name, updatePriority.ResourceDiff, getProcessedRecommendationTargets(processedRecommendation))
```

Comment:
This is now a method on `calc`

```suggestion
	klog.V(2).Infof("pod accepted for update %v/%v with priority %v - processed recommendations:\n%v", pod.Namespace, pod.Name, updatePriority.ResourceDiff, calc.GetProcessedRecommendationTargets(processedRecommendation))
```

Reply:
ah, yes! I reverted some changes on this line while committing that also reverted changing it to a `calc` method 🙈 

fixed, thank you! ✅